### PR TITLE
refactor(theme): menus icons depending on theme choosing refactor

### PR DESCRIPTION
**Pull request by 32penkin**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/kittenTricks/blob/master/CONTRIBUTING.md) guide.

### CODE REVIEW

File: src/containers/menu/components/routes.ts [hunk 1]
```diff
@@ -33,110 +33,156 @@ import {
   ComponentsIconBottomNavigationDark,
 } from '@src/assets/icons';
 import { ThemeKey } from '@src/core/themes';
+import { MenuImage } from '../menuImage';
```
Comment (artyorsh) [inline src/containers/menu/components/routes.ts]: refactor this to `@src/core/themes` utility
like `Theme.select`

File: src/containers/menu/menuImage.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+export class MenuImage {
+
+  public static select = (config: LayoutMenuImageConfig,
+                          currentTheme: ThemeKey): any => {
+
+    const currentKey: string = Object.keys(config)
+      .find((key: string) => key === currentTheme);
+
+    return config[currentKey];
```
Comment (artyorsh) [inline src/containers/menu/menuImage.ts]: :)
```ts
return config[currentTheme]
```

File: src/containers/menu/menuImage.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { ImageProps } from 'react-native';
+import { ThemeKey } from '@src/core/themes';
+
+export type LayoutMenuImageConfig = { [key in ThemeKey]?: React.ReactElement<ImageProps> };
+
+export class MenuImage {
+
+  public static select = (config: LayoutMenuImageConfig,
```
Comment (artyorsh) [inline src/containers/menu/menuImage.ts]: Should be geneneric which returns Generic or null

Comment (artyorsh) [reply src/containers/menu/menuImage.ts]: Also makes sense to return default value if has no value found in config:
```ts
select({
  'light': foo,
  'default': bar,
});
```
Take a look on React Native's `Platform.select` function

**Review by artyorsh — APPROVED** [review]

File: src/containers/menu/components/routes.ts (hunk 1 above)
Comment (32penkin) [reply src/containers/menu/components/routes.ts]: Done.

File: src/containers/menu/menuImage.ts (hunk 2 above)
Comment (32penkin) [reply src/containers/menu/menuImage.ts]: Done.

File: src/containers/menu/menuImage.ts (hunk 3 above)
Comment (32penkin) [reply src/containers/menu/menuImage.ts]: Done.

**Pull request merged**